README.md

```diff
@@ -4,6 +4,8 @@
 $ yarn install
 ```
 
+File public/app-config.json must be created by duplicating public/app-config.template.json and filling in the encryptionKey.
```

Comment:
[minor] Maybe some backticks on files and keys to highlight them: `public/app-config.js`, and so on